ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/ccj/CountyCourtJudgmentMapper.java

```diff
@@ -17,13 +17,17 @@
 import static uk.gov.hmcts.cmc.ccd.domain.CCDPaymentOption.valueOf;
 
 @Component
-public class CountyCourtJudgmentMapper implements Mapper<CCDCountyCourtJudgment, CountyCourtJudgment> {
+public class CountyCourtJudgmentMapper {
 
-    @Override
-    public CCDCountyCourtJudgment to(CountyCourtJudgment countyCourtJudgment) {
+    public CCDCountyCourtJudgment to(Claim claim) {
+
```

Comment:
in all mapper we are checking for        
        ```Objects.requireNonNull(claim, "claim must not be null");```

Reply:
we are returning null if we cant get details. So I am returning null if either claim or claim.getCCJ is null.

Reply:
Claim to asserted null should be part of claimmapper.